tests/address_bar_and_search/test_suggestion_engine_selection.py

```diff
@@ -24,6 +24,7 @@ def test_search_suggestion_for_engine_selection(driver: Firefox, site: str):
     nav.type_in_awesome_bar("@")
     nav.element_has_text("results-dropdown", f"Search with {site}")
 
+    nav.set_chrome_context()
```

Comment:
Is there are reason to do this vs. using the context manager?

Reply:
the context manager didnt work for some reason